swift/apple/FirezoneKit/Sources/FirezoneKit/Managers/TunnelManager.swift

```diff
@@ -281,7 +282,7 @@ public class TunnelManager {
           self.resourceListHash = Data(SHA256.hash(data: data))
           let decoder = JSONDecoder()
           decoder.keyDecodingStrategy = .convertFromSnakeCase
-          self.resourcesListCache = (try? decoder.decode([Resource].self, from: data)) ?? []
+          self.resourcesListCache = ResourceList.loaded(try! decoder.decode([Resource].self, from: data))
```

Comment:
I made this panic if the decode fails, hopefully that's fine?

Reply:
Yeah, given that connlib is embedded, we don't need to worry about backwards-compatibility here so this either works always or never.